roles/confluent.kafka/templates/server_sasl_ssl.properties.j2

```diff
@@ -1,5 +1,5 @@
 # Maintained by Ansible
-listeners=SASL_SSL://:{{broker.config.port}}
+listeners=SASL_SSL://:{{kafka_port}}
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.